pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdTopics.java

```diff
@@ -1336,11 +1335,35 @@ void run() throws PulsarAdminException {
                 , description = "ManagedLedger offload deletion lag in bytes")
         private Long offloadDeletionLagInMillis;
 
+        @Parameter(
+                names = {"--offloadedReadPriority", "-orp"},
+                description = "read priority for offloaded messages",
+                required = false
+        )
+        private String offloadReadPriorityStr;
+
         @Override
         void run() throws PulsarAdminException {
             String persistentTopic = validatePersistentTopic(params);
+
+            var offloadedReadPriority = OffloadPolicies.DEFAULT_OFFLOADED_READ_PRIORITY;
```

Comment:
not suggest to use `var`

Reply:
@hangc0276 
The official support of type inference in Java began at Java 10, I think it's a good feature to keep code base concise, before Java 10, we can use the Lombok version first, just like we use JodaTime before Java8.

https://openjdk.java.net/jeps/286

Reply:
Let's avoid using `var` now.